ui/widgets/calendar.js

```diff
@@ -150,7 +151,14 @@ return $.widget( "ui.calendar", {
 		}
 
 		if ( this._needsRefresh() ) {
-			this._refresh();
+			if ( this.options.numberOfMonths > 1 && this.date.year() === this.viewDate.year() ) {
+				this.viewDate.adjust( "M", this.options.numberOfMonths *
+					( this.date.month() > this.viewDate.month() ? 1 : -1 )
+				);
+				this.refresh();
+			} else {
+				this._refresh();
```

Comment:
Mhh, not sure if that makes sense. Do we want a method (perhaps "_updateView") with only one line of code in it? 
Perhaps we should remove it completely (already thought about this) and just add the additional line four times. Or just rename the method.